public/javascripts/payments.js

```diff
@@ -525,7 +545,7 @@
     start = null
   ) => {
     start = start ? start : Date.now();
-    const endStates = ['succeeded', 'processing', 'canceled'];
+    const endStates = ['succeeded', 'processing', 'canceled', 'requires_payment_method'];
```

Comment:
🤔 When a P24 payment fails when the customer does not authenticate the payment, the `PaymentIntent` will transition to `requires_payment_method`. In this case, we don't want to continue polling on the page, as the `status` will not change.

Is this the correct handling?

Reply:
Good catch, just realized this also already affects iDEAL, so good that get that in, thanks!

Reply:
Good catch, just realized this also already affects iDEAL, so good that get that in, thanks!